user-interface/src/main/java/life/qbic/datamanager/views/projects/project/info/ProjectSummaryComponent.java

```diff
@@ -530,14 +533,18 @@ private void handleUpdateEvent(ProjectDesignUpdateEvent event) {
     var request = new ProjectUpdateRequest(project,
         new ProjectDesign(info.getProjectTitle(), info.getProjectObjective()));
 
-    asyncProjectService.update(request)
+    submit(request, req -> asyncProjectService.update(req)
```

Comment:
```suggestion
    submit(request, requestConsumer -> asyncProjectService.update(requestConsumer)
```

Reply:
where do you make use of the request itself here?
sumbit takes a request as first parameter and a consumer as second. Maybe I misunderstood something here?